client/helper.go

```diff
@@ -167,3 +168,21 @@ func ShowJSONResponse(resp *resty.Response) (success bool) {
 func GetRequest() *resty.Request {
 	return GetRequestTimeout(DefaultTimeout)
 }
+
+// Streams out a text response, like ones from /host/logs
+func StreamTextResponse(resp *resty.Response) (success bool) {
+	success = true
+	for {
+		p := make([]byte, 4096)
+		n, err := resp.RawBody().Read(p)
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			fmt.Println(err)
+			success = false
+			break
+		}
+		fmt.Print(string(p[:n]))
+	}
+	return
+}
```

Comment:
Hm, this isn't working. Testing it now, I don't always get the expected number of results based on range. The API itself seems to be working fine when I hit it `curl` in testing so I believe the problem is in here. Looking into it now.

Reply:
All good now. Thanks @agners !